posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -258,8 +272,8 @@ and
 [source,java]
 ----
 Config config = ConfigProvider.getConfig();
-config.getValue("empty.property", String.class); // Will thorw a `NoSuchElementException`
-config.getValue("empty.array.property", String[].class); // Will thorw a `NoSuchElementException`
+config.getValue("empty.property", String.class); // Will throw a `NoSuchElementException`
```

Comment:
Change,
"config.getValue("empty.property", String.class); // Will throw a `NoSuchElementException`"
to,
"config.getValue("empty.property", String.class); // throws a `NoSuchElementException`"